### ISSUE TITLE
DISSIDIA FINAL FANTASY NT Free Edition (921590)

### ISSUE THREAD

**Issue by Anupalabdhi** [ValveSoftware/Proton#2410]
# Compatibility Report
- Name of the game with compatibility issues: DISSIDIA FINAL FANTASY NT Free Edition
- Steam AppID of the game: 921590

## System Information
- GPU: RX 580
- Driver/LLVM version: Mesa 18.3.4 
- Kernel version: 5.0.0 
- Link to full system information report as [Gist](https://gist.github.com/Anupalabdhi/83c526e0b0d4c100bf8e2812c024a305/):
- Proton version: 3.16-8

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-921590.log](https://github.com/ValveSoftware/Proton/files/2958569/steam-921590.log)

## Symptoms  
The game launches to a blank white screen. The game is running in the background, you can hear the audio and menu progresses based on inputs sent.

## Reproduction
Step 1: Launch the game
Step 2: White Screen instead of actual menu/game

Edit:
Apparently I was able to maneuver my way into an actual online matchup and the white screen display is still present as ever so this is not just a matter of menu not displaying. Also cutscenes are being skipped so mfplat also might be an issue here.

**Comment by witcheslive** [comment]
I just tried this for the first time and got the white screen. It seems to be working otherwise, I can hear menu selections, but I can't see anything.

**Comment by masterzorag** [comment]
![Screenshot from 2019-05-01 12-50-39](https://user-images.githubusercontent.com/8250079/57039943-5a58b780-6c5e-11e9-8e5d-82307cd5c057.png)
Proton-4.2-3 shipped by Steam reach and past main title, but now crashes on its wine-4.2
also, https://www.winehq.org/announce/4.4 lists some MFPlat progress...

_fedora 29, mesa-vulkan-drivers-18.2.4-2.fc29.x86_64_

**Comment by ahjolinna** [comment]
for me on latest proton 4.2-3 it does seems to run without any crashes but the performance is just awful even with all settings at low/off.

------
system spec:

```
inxi -bx
System:    Host: linux Kernel: 5.0.11-1-default x86_64 bits: 64 compiler: gcc v: 8.3.1 Desktop: KDE Plasma 5.15.4 
           Distro: openSUSE Tumbleweed 20190508 
Machine:   Type: Desktop Mobo: ASUSTeK model: Z170 PRO GAMING v: Rev X.0x serial: <root required> UEFI: American Megatrends 
           v: 3805 date: 05/16/2018 
Battery:   Device-1: sony_controller_battery_90:fb:a6:df:fa:93 model: N/A charge: N/A status: Full 
CPU:       Quad Core: Intel Core i5-6600K type: MCP arch: Skylake-S speed: 4374 MHz min/max: 800/3900 MHz 
Graphics:  Device-1: NVIDIA GM204 [GeForce GTX 970] vendor: eVga.com. driver: nvidia v: 430.09 bus ID: 01:00.0 
           Display: x11 server: X.Org 1.20.4 driver: nvidia resolution: 1920x1080~60Hz, 1920x1080~60Hz 
           OpenGL: renderer: GeForce GTX 970/PCIe/SSE2 v: 4.6.0 NVIDIA 430.09 direct render: Yes 
Network:   Device-1: Intel Ethernet I219-V vendor: ASUSTeK driver: e1000e v: 3.2.6-k port: f000 bus ID: 00:1f.6 
Drives:    Local Storage: total: 34.11 TiB used: 33.18 TiB (97.3%) 
Info:      Processes: 392 Uptime: 13h 59m Memory: 15.60 GiB used: 9.40 GiB (60.2%) Init: systemd runlevel: 5 Compilers: 
           gcc: 8.3.1 Shell: bash v: 5.0.7 inxi: 3.0.32
```

**Comment by Wedge009** [comment]
I gave this a try a few weeks ago - Proton 6.3 with default configuration. Everything seemed to work fine but I only played one round. I can't comment on how performance compares with an equivalent Windows system but subjectively I do feel as though there might be a performance disparity. Better than not working at all, I suppose.

**Comment by Wedge009** [comment]
Something seems to have broken for me with Proton 8.0-3 - I made a similar report in #585. Using Proton 8.0-2d from @alasky17, I made these two logs for comparison. My system is still https://gist.github.com/Wedge009/2e9244b65fb276eab4e79897785669a6

[steam-921590-proton-8.0-2d.log](https://github.com/ValveSoftware/Proton/files/12160951/steam-921590-proton-8.0-2d.log)
[steam-921590-proton-8.0-3.log](https://github.com/ValveSoftware/Proton/files/12160952/steam-921590-proton-8.0-3.log)

The game plays as normal with 8.0-2d but fails to start in 8.0-3.

**Comment by Wedge009** [comment]
As with #585, using the older dxvk from Proton 8.0-2, in Proton 8.0-3 seem to allow this game to run again. So it appears to be a problem with the dxvk dependency.

**Comment by alasky17** [comment]
Tagging @doitsujin @Blisto91 on this one too.  @Wedge009 Thank you for checking this!  Seems like something in the dxvk update in 8.0-3 really did not like your setup for some reason.  Did you try Proton - Experimental yet to confirm that the regression is not already fixed upstream?

**Comment by Wedge009** [comment]
Yes, for both this and #585, Proton Experimental (as at 2023-07-27 00:55 UTC) doesn't appear to have any change from Proton 8.0-3. Neither allow these two games to run, on my machine.

**Comment by alasky17** [comment]
@Wedge009 Thank you for checking :) I figured it was worth a shot :D

**Comment by Wedge009** [comment]
There was an update for Proton Experimental. But as it turns out, I think my issue was more to do with my storage as I confirmed separately (see https://github.com/doitsujin/dxvk/issues/3584 for the ugly details) that the issue is not related to dxvk.

@kisak-valve You can remove the Regression label. Sorry for wasting everyone's time.
